cmake/CMakeLists.txt

```diff
@@ -1,15 +1,18 @@
 # Minimum CMake required
 cmake_minimum_required(VERSION 2.8.12)
 
-# Project
-project(protobuf C CXX)
-
 # CMake policies
 cmake_policy(SET CMP0022 NEW)
 
+# Project
+project(protobuf C CXX)
+
 # Options
 option(protobuf_VERBOSE "Enable for verbose output" OFF)
+option(protobuf_TARGETS_ALIASES "Provide targets aliases without protobuf:: namespace" ON)
+option(protobuf_MODULE_COMPATIBLE "CMake build-in FindProtobuf.cmake module compatible" ON)
```

Comment:
If it *is* provided, it should be moved to protobuf-config.in.
config files are only parsed if the version is accepted, so it's perfectly safe and much more helpful to the user.